src/framework/preferences/js/Panels.js

```diff
@@ -281,8 +281,9 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
      * Only use in cases where the instatiated component cannot be used.
      */
     fluid.prefs.compositePanel.prefetchComponentOptions = function (type, options) {
-        var baseOptions = fluid.getGradedDefaults(type, fluid.get(options, "gradeNames"));
-        return fluid.merge(baseOptions.mergePolicy, baseOptions, options);
+        var baseOptions = fluid.getMergedDefaults(type, fluid.get(options, "gradeNames"));
+        // TODO: awkwardly, fluid.merge is destructive on each argument!
```

Comment:
It will tend to corrupt them in various hard-to-predict ways, for example by adding properties whose value is `undefined` or replacing IoC references with their targets. As our docs say, fluid.merge should not be called by users - http://docs.fluidproject.org/infusion/development/OptionsMerging.html . In the next framework rewrite we might improve this. In any case, this update was helpful in revealing this fact - without the fluid.copy(), we now get an error triggered as fluid.merge tries to corrupt the defaults record which is now immutable.